src/index.js

```diff
@@ -33,16 +38,28 @@ module.exports = options => ({
   },
   // Fix sourcemaps
   result(stylelintResult, filepath) {
+    const interpolationLines = interpolationLinesMap[filepath] || []
     const lineCorrection = sourceMapsCorrections[filepath]
-    const warnings = stylelintResult.warnings.map(warning =>
-      Object.assign({}, warning, {
-        // Replace "brace" with "backtick" in warnings, e.g.
-        // "Unexpected empty line before closing backtick" (instead of "brace")
-        text: warning.text.replace(/brace/, 'backtick'),
-        line: lineCorrection[warning.line]
-      })
-    )
+    const warnings = stylelintResult.warnings
+      .filter(
+        warning =>
+          // Filter indentation warnings generated by interpolations substitution
+          warning.rule !== 'indentation' ||
+          interpolationLines.indexOf(lineCorrection[warning.line]) < 0
+      )
+      .map(warning =>
+        Object.assign({}, warning, {
+          // Replace "brace" with "backtick" in warnings, e.g.
+          // "Unexpected empty line before closing backtick" (instead of "brace")
+          text: warning.text.replace(/brace/, 'backtick'),
+          line: lineCorrection[warning.line]
+        })
+      )
 
-    return Object.assign({}, stylelintResult, { warnings })
+    return Object.assign({}, stylelintResult, {
+      warnings,
+      // Remember to fix
```

Comment:
Is this a note to yourself to remember something, or is it an explanatory comment? If it's an explanatory comment maybe make it a bit more verbose, such as `// Undo errored if all warnings were caused by interpolation substitutions`